index.bs

```diff
@@ -6825,11 +6826,13 @@ An <dfn id="dfn-observable-array-type" export>observable array type</dfn> is a p
 whose values are references to a combination of a mutable list of objects of type |T|, as well as
 behavior to perform when author code modifies the contents of the list.
 
-The parameterized type must not be a [=dictionary type=], [=sequence type=], or [=record type=].
+The parameterized type must not be a [=dictionary type=], [=sequence type=], [=record type=], or [=observable array type=].
 
 Similar to [=sequence types=] and [=frozen array types=], observable array types wrap around
 ECMAScript array types, imposing additional semantics on their usage.
 
+Observable array types are non-nullable, but T may be nullable.
```

Comment:
I'd suggest removing this sentence, and have the nullability of |T| moved up. It's not that observable array types themselves are non-nullable (every type other than nullable types are non-nullable), but more like they cannot be the inner type of a nullable type.

Reply:
Actually, I see the same sentence is used for promise types. Consider this optional then.

Reply:
Yeah, I saw the same sentence is used for promise types, so I added this.
But I think you are right. And the nullable type section already mentions inner type could not be an observable array type, it should be clear enough.